frontend/pages/admin/IntegrationsPage/cards/MdmSettings/AppleBusinessManagerPage/components/AppleBusinessManagerTable/AppleBusinessManagerTableConfig.tsx

```diff
@@ -163,12 +164,20 @@ export const generateTableConfig = (
       // but we don't use it.
       accessor: "id",
       Cell: (cellProps) => (
-        <ActionsDropdown
-          options={generateActions()}
-          onChange={(value: string) =>
-            actionSelectHandler(value, cellProps.row.original)
-          }
-          placeholder="Actions"
+        <GitOpsModeTooltipWrapper
+          position="left"
+          renderChildren={(disableChildren) => (
+            <div className={disableChildren ? "disabled-by-gitops-mode" : ""}>
```

Comment:
Is the additional `div` and `disabled-by-gitops-mode` class needed?

I think the disabled `ActionsDropdown` will already have a disabled class applied to it and we can use this styling logic on it:

```
  .actions-dropdown-select {
    &--is-disabled {
      @include disabled;
    }
    &__control--is-disabled {
      min-height: 0;
      padding: 0;
    }
  }
```

Reply:
removed it and yes it seems its requires still.